app/views/receipt_mailer/receipt_email.html.erb

```diff
@@ -1,28 +1,44 @@
-<p>Hello,</p>
+<p>Hello <%= @user.given_name %>,</p>
 
-<p>You are receiving this message to confirm your successful submission of
-either your ProQuest form, title page, and abstract or the electronic version of
-your <%= @thesis.graduation_month %> <%=
-@thesis.graduation_year %> thesis "<%= @thesis.title %>" to the MIT Libraries.
-</p>
+<p>You have successfully updated your thesis information. 
+Here is the information we have on record:</p>
 
-<p>If you submitted a May or September 2020 thesis, please note that only the
-ProQuest form, title page, and abstract should have been submitted here. The
-actual thesis should go directly to your department. For degree periods prior to
-May 2020, we accept all electronic copies via this form.</p>
+<ul>
+  <li>Name: <%= @user.display_name %></li>
+  <li>Email: <%= @user.email %></li>
+  <li>ORCID iD: <%= @user.orcid %></li>
+  <li>Thesis title: <%= @thesis.title %></li>
+  <li>Name as it appears on your thesis: <%= @user.preferred_name %></li>
+  <li>Co-author: <%= @thesis.coauthors %></li>
+  <li>Department(s): <%= @thesis.departments.map { |d| d.name_dw }.join(', ') %></li>
+  <li>Degree(s): <%= @thesis.degrees.map { |d| d.name_dw }.join(', ') %></li>
+  <li>Degree date: <%= @thesis.graduation_month %> <%= @thesis.graduation_year %></li>
+  <li>Copyright holder: <%= @thesis.copyright.holder %></li>
+  <li>License: <%= @thesis.license.display_description %></li>
+  <li>Thesis supervisors: <%= @thesis.advisors.map { |a| a.name }.join(', ') %></li>
+  <li>Abstract: <%= @thesis.abstract %></li>
+  <li>Notes: <%= @thesis.author_note %></li>
+</ul>
 
-<p>You do not need to reply to this message.</p>
+<p>You have until graduation day to update this information.</p>
 
-<p>Theses will be added to DSpace after degrees have been awarded and the thesis
-has been fully processed by the MIT Libraries (typically 3-6 months after the
-degree date).</p>
+<p>A few reminders:</p>
+<ul>
+  <li>Save your files in the correct format.</li>
+    <ul>
```

Comment:
I'm not sure how finicky HTML rendering is within email, but I don't think this markup is valid - the inner list should be inside a list item element according to the HTML spec. That said, HTML in emails can be weird - so if you did this because of a quirk there, then nevermind me.

Reply:
Oh, good catch! I was on autopilot while marking that up and missed that one.

Reply:
Having now sent emails to myself via gmail and to MIT (which is where I'm guessing most of these emails will be rendered), I see the nesting appears correctly - so I'm less worried about this issue.